src/hardware/src/rover_arm.cpp

```diff
@@ -58,36 +63,102 @@ hardware_interface::CallbackReturn RoverArmHardwareInterface::on_init(
     }
   }
 
-  node_ = rclcpp::Node::make_shared("get_angle_client");
-  client_ = node_->create_client<interfaces::srv::ArmPos>("arm_pos");
+  node_ptr_ = std::make_shared<rclcpp::Node>("arm_hw_subscriber");
```

Comment:
```suggestion
  node_ptr_ = rclcpp::Node::make_shared("arm_hw_subscriber");
```

Technically does the same thing but for consistancy